lessons/find_number.rb

```diff
@@ -0,0 +1,26 @@
+secret_number = rand(1..100)
+
+puts 'Guess the number'
+number_guess = 0
+
+while true
+    puts "Enter number from 1 to 100"
+    number = gets.to_i
+    number_guess += 1
+    if number == secret_number
+        puts "You guessed the number #{secret_number} in #{number_guess} try/tries."
+        puts "Do you want to play again? Print Yes or No"
+        if gets.chomp.upcase == "Yes"
+            secret_number = rand(1..100)
+            number_steps = 0
+            next
+        else
+            puts "Good Bye!"
+            break
+        end
+    elsif number > secret_number
+        puts "Too big! Try the smaller one."
+    else
+        puts "Too small! Try the bigger one."
```

Comment:
❗ If I input something that is not a number, the program will say my guess was too small, but it should instead tell me my guess was invalid.

The `#to_i` method converts non-numbers to `0` so, can you add a condition to handle this scenario: tell the user their input was invalid and request they enter a number between 1-100?